tests/components/music_assistant/test_config_flow.py

```diff
@@ -0,0 +1,191 @@
+"""Define tests for the Music Assistant Integration config flow."""
+
+from copy import deepcopy
+from ipaddress import ip_address
+from unittest import mock
+
+from music_assistant.client.exceptions import CannotConnect, InvalidServerVersion
+
+from homeassistant.components.music_assistant import config_flow
+from homeassistant.components.music_assistant.config_flow import CONF_URL
+from homeassistant.components.music_assistant.const import DOMAIN
+
+# pylint: disable=wrong-import-order
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+from homeassistant.config_entries import SOURCE_USER, SOURCE_ZEROCONF
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+DEFAULT_TITLE = "Music Assistant"
+
+VALID_CONFIG = {
+    CONF_URL: "http://localhost:8095",
+}
+VALID_OPTIONS_CONFIG = {
+    CONF_URL: "http://localhost:8095",
+}
+
+SERVER_INFO = {
+    "server_id": "1234",
+    "base_url": "http://localhost:8095",
+    "server_version": "0.0.0",
+    "schema_version": 23,
+    "min_supported_schema_version": 23,
+    "homeassistant_addon": True,
+}
+
+ZEROCONF_DATA = ZeroconfServiceInfo(
+    ip_address=ip_address("127.0.0.1"),
+    ip_addresses=[ip_address("127.0.0.1")],
+    hostname="mock_hostname",
+    port=None,
+    type=mock.ANY,
+    name=mock.ANY,
+    properties={
+        "server_id": "1234",
+        "base_url": "http://localhost:8095",
+        "server_version": "0.0.0",
+        "schema_version": 23,
+        "min_supported_schema_version": 23,
+        "homeassistant_addon": True,
+    },
+)
+
+
+# pylint: disable=dangerous-default-value
```

Comment:
I agree, we should not have VALID_CONFIG as default parameter to avoid strange stuff

Reply:
Removed